discretisedfield/field.py

```diff
@@ -693,12 +685,7 @@ def mean(self, direction=None):
 
     @property
     def average(self):
-        warnings.warn(
-            "The average property is deprecated. Please use the mean function.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        return self.mean()
+        raise ValueError("This property has been remove, please use the mean function.")
```

Comment:
If it is a method, other wise property?
```suggestion
        raise ValueError("This property has been remove, please use the mean method.")
```